assets/src/selected-stories-block/block/storyPreview.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { CardPreviewContainer } from '../../dashboard/components';
```

Comment:
Hi @ediamin , just a thought, should we have these components in this block's folder separately? So that in future in case they get updated, it doesn't break our block?

Reply:
I don't think that's a good idea. We should always reuse any component that we can.

Reply:
I understood. But will this not add a dependency for that component. In future if someone decides to make changes in that component(s), they will have to factor in supporting/updating our block as well.

Reply:
Yes, of course. That's a pitfall of using reusing single component across the project.

Reply:
Ok noted 👍  Thanks!